packages/cli-platform-ios/src/commands/buildIOS/buildProject.ts

```diff
@@ -33,8 +34,8 @@ export function buildProject(
       (udid
         ? `id=${udid}`
         : mode === 'Debug'
-        ? 'generic/platform=iOS Simulator'
-        : 'generic/platform=iOS') +
+        ? `generic/platform=${platform} Simulator`
+        : `generic/platform=${platform}`) +
```

Comment:
since this is `iOS` not `ios`, maybe we should have a `humanReadablePlatformName` variable or something to disambiguate?

Reply:
also, `iOS Simulator` shouldn't be concatenated, makes it harder to search for strings in the repo

Reply:
I've added humanReadablePlatformName, but I still need to sort out the concatenation to make it more reusable